### DISCUSSION TITLE
Is there a way to unlock a random scenario?

### DISCUSSION THREAD

**Discussion by mogul** [Lurkars/gloomhavensecretariat#560] [Q&A]
We need to unlock a random scenario, but I'm not seeing a way to do that. Am I missing it? Or are we expected to draw a card from the box, and then unlock that drawn scenario's number in the app?

**Comment by Lurkars** [comment]
When not playing scenario, there should be "Event Effects" menu next to the level adjustment on the bottom. If you open it, there should be a button "Draw Random Scenario"

GH:
![image](https://github.com/Lurkars/gloomhavensecretariat/assets/2760570/14b75669-9022-4652-b1f1-ba3249022c9a)

FH:
![image](https://github.com/Lurkars/gloomhavensecretariat/assets/2760570/bf7f9d43-6965-43fd-b1a0-c08c80db4c3d)
